test/blackbox-tests/test-cases/package-version-empty.t/run.t

```diff
@@ -0,0 +1,9 @@
+Testing dune #10674 where an empty version in an opam file caused a code error
+in dune. We should make sure that this case is handled gracefully.
+
+  $ dune build
+  File "foo.opam", line 2, characters 0-7:
+  2 | version: ""
+      ^^^^^^^
+  Error: "" is an invalid package version.
```

Comment:
In the future, perhaps you could tell the user what's a valid version. E.g. a non empty string.